src/analysis/retail/monk/mistweaver/normalizers/CastLinkNormalizer.ts

```diff
@@ -619,4 +679,20 @@ export function HasStackChange(event: RefreshBuffEvent): boolean {
   return HasRelatedEvent(event, MT_STACK_CHANGE);
 }
 
+export function atFromEssenceFont(event: ApplyBuffEvent | RefreshBuffEvent) {
+  return HasRelatedEvent(event, ANCIENT_TEACHINGS_EF);
+}
+
+export function atFromFaelineStomp(event: ApplyBuffEvent | RefreshBuffEvent) {
+  return HasRelatedEvent(event, ANCIENT_TEACHINGS_FLS);
+}
```

Comment:
I would prefer we call the functions `isAtFromX` to maintain parity with other functions